src/rust/ide/enso-protocol/src/file_manager.rs

```diff
@@ -67,24 +74,37 @@ pub enum Notification {
 #[derive(Clone,Debug,PartialEq)]
 #[derive(Serialize, Deserialize)]
 pub struct FilesystemEvent {
-    /// Path of the file that the event is about.
-    pub path : Path,
     #[allow(missing_docs)]
+    // The serialization format requires the information to be wrapped around an "event" property.
```

Comment:
Plus this documentation is semantically wrong. It doesn't tell anything about this field - it tells something about serialization format, not this field per se.

Reply:
Oh, ok, good point!